stock_available_sale/views/product_template_view.xml

```diff
@@ -3,8 +3,8 @@
     <data>
         <!-- Add the quantity available to promise in the product form -->
         <record id="view_product_form_quoted_qty" model="ir.ui.view">
-            <field name="name">product.form.quoted_qty</field>
-            <field name="model">product.product</field>
+            <field name="name">Quoted quantity on product form</field>
+            <field name="model">product.template</field>
```

Comment:
If I'm not mistaken, this view is reused for both product templates and variants so doing it on product.template serves both purposes at once.
The tree view on the other hand seems to have been specifically inherited for product.product in the standard, but I'm not adding the new field there.